charts/README.md

```diff
@@ -123,4 +122,3 @@ helm delete `<RELEASE NAME>` -n `<RELEASE NAMESPACE>`
 | release.version | string | `"4.0.1"` |  |
 | zfs-localpv.crds.csi.volumeSnapshots.enabled | bool | `false` |  |
 | zfs-localpv.crds.zfsLocalPv.enabled | bool | `true` |  |
-| zfs-localpv.enabled | bool | `true` |  |
```

Comment:
why are these 3 removed? Are they redundant?

Reply:
These options are no longer valid. They were removed during v4.0.0 release. However, this README.md still documents them.

Not related to the change of this PR.